### ISSUE TITLE
Visual Glitch

### ISSUE THREAD

**Issue by Notlucky101** [V0idWa1k3r/SmartBackpacks#38]
though the items are in the backpack, I have to click on any spot to see them when initially open the backpack.

**Comment by davidsielert** [comment]
yeah this is definitely the problem . thanks for at least pointing it out .. needs a re-render

**Comment by Algalish** [comment]
I also have this problem in the last version. Very hard to use it with this bug.
